lib/input_parser.ex

```diff
@@ -0,0 +1,78 @@
+defmodule InputParser do
+  @exit_command "exit"
+  @flag_command "-f"
+
+  def exit_command, do: @exit_command
+
+  def parse_turn(input) do
+    find_turn_input_type(input)
+    |> parse()
+  end
+
+  def parse_number(input) do
+    cond do
+      is_exit_command?(input) ->
+        {:exit, input}
+
+      true ->
+        {:number, input}
+    end
```

Comment:
I want to check that the input is actually a number, but that will happen in a later PR for missing validations